packages/bindgen/package.json

```
@@ -13,7 +13,8 @@
     "test": "mocha",
     "generate:spec-schema": "typescript-json-schema tsconfig.json RelaxedSpec --include src/spec/relaxed-model.ts --out generated/spec.schema.json --required",
     "build": "cmake-js build --debug",
-    "rebuild": "cmake-js rebuild --debug"
+    "rebuild": "cmake-js rebuild --debug",
+    "build-browser": "emcmake cmake -B build.browser -G Ninja && cd build.browser && ninja"
```

```suggestion
    "build-browser": "emcmake cmake -B build.browser -G Ninja && cmake --build build.browser"
```

There are some benefits to using `cmake --build` if we switch to using `-G "Ninja Multi-Config" which I think we will want to (but not quite yet).